Content.Server/SS220/SpaceWars/Party/IPartyManager.cs

```diff
@@ -0,0 +1,70 @@
+
+using Content.Shared.SS220.SpaceWars.Party;
+using Robust.Shared.Console;
+using Robust.Shared.Player;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Content.Server.SS220.SpaceWars.Party;
+
+public partial interface IPartyManager : ISharedPartyManager
+{
+    event Action<PartyStatusChangedActionArgs>? PartyStatusChanged;
+    event Action<Party>? PartyUpdated;
+
+    event Action<PartyMember>? UserJoinedParty;
+    event Action<PartyMember>? UserLeavedParty;
+
+    IReadOnlyCollection<Party> Parties { get; }
+
+    bool TryCreateParty(ICommonSession host, [NotNullWhen(true)] out Party? party, PartySettingsState? settings = null, bool force = false);
+
+    Party CreateParty(ICommonSession host, PartySettingsState? settings = null, bool force = false);
+
+    void DisbandParty(Party party, bool updates = true);
+
+    bool TryAddMember(Party party,
+        ICommonSession session,
+        PartyMemberRole role = PartyMemberRole.Member,
+        bool force = false,
+        bool ignoreLimit = false,
+        bool updates = true,
+        bool notify = true);
+    void AddMember(Party party,
+        ICommonSession session,
+        PartyMemberRole role = PartyMemberRole.Member,
+        bool force = false,
+        bool ignoreLimit = false,
+        bool updates = true,
+        bool notify = true);
+
+
+    bool TryRemoveMember(Party party, ICommonSession session, bool updates = true, bool notify = true);
+    void RemoveMember(Party party, ICommonSession session, bool updates = true, bool notify = true);
```

Comment:
```suggestion
        bool notify = true);

// Remove Member

    bool TryRemoveMember(Party party, ICommonSession session, bool updates = true, bool notify = true);
    
    void RemoveMember(Party party, ICommonSession session, bool updates = true, bool notify = true);
```
Раздели просто комментарием, хотя я считаю что такое деление не нужно.